cmd/zpios/zpios_main.c

```diff
@@ -551,6 +551,7 @@ run_one(cmd_args_t *args, uint32_t id, uint32_t T, uint32_t N,
 
 	if (args->verbose) {
 		rc2 = read(zpiosctl_fd, zpios_buffer, zpios_buffer_size - 1);
+		zpios_buffer[zpios_buffer_size - 1] = '\0';
```

Comment:
This buffer should always be zero terminated because it's initialized to all zeros and then always `zpios_buffer_size-1` characters are read.  Adding this line is fine by then we should update the read length above to be `zpios_buffer_size`.